### PR TITLE
Fix latest meeting button

### PR DESCRIPTION
**Pull request by cydrobolt**
#115

### CODE REVIEW

**Comment by cydrobolt** [comment]
@pypingou could you review this?

File: mote/__init__.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 import flask, random, string, json, util, re
 import dateutil.parser, requests, collections
-import logging
+import logging, soke
```
Comment (pypingou) [inline mote/__init__.py]: This doesn't seem to be used in the diff here

File: mote/__init__.py [hunk 2]
```diff
@@ -345,7 +345,7 @@ def sresults():
         type = group_type,
         avail_dates = avail_dates,
         meetbot_location = config.meetbot_prefix,
-        latest_meeting = next(reversed(avail_dates.items()[0][1].items()[-1][1]))
+        latest_meeting = (avail_dates.items()[0][1].items()[0][1])[0]
```
Comment (pypingou) [inline mote/__init__.py]: I don't think you need the brackets, do you?

Comment (cydrobolt) [inline mote/__init__.py]: You're right, I don't think we need the parentheses.

File: mote/__init__.py (hunk 1 above)
Comment (cydrobolt) [inline mote/__init__.py]: The import is used elsewhere in the file, but it's never caused problems even though it was never imported before. 

pyflakes was complaining about the missing import, so I added it. I'm not sure why it was working before even though the import was missing. Maybe you have a better idea of why?

Comment (pypingou) [inline mote/__init__.py]: Because it seems to be only used in Except clauses, so if you never faced the exception, it was never called :)

Comment (cydrobolt) [inline mote/__init__.py]: Interesting, thanks for clearing that up for me :)
We should probably add the import. I remember someone mentioning that random 500s happened occasionally. Maybe those could have been related to this issue.

Comment (pypingou) [inline mote/__init__.py]: We should definitively keep the import since otherwise we are creating an error inside a exception handling :)

**Comment by cydrobolt** [comment]
I removed the unnecessary parentheses. Thanks @pypingou!

**Pull request merged**